### DISCUSSION TITLE
QFieldCloud Support Tickets

### DISCUSSION THREAD

**Discussion by Oisin-JK** [opengisch/QField#4428] [Q&A]
HI. I posted a support ticket on QFieldCloud website via the ? button.  Is there a way of viewing the ticket after it has been submitted: (1) to check it has been submitted; and (2) to see if there has been any progress.?
Thanks

**Comment by GonLes** [comment]
Hello @Oisin-JK , after you submit the ticket you will receive an email, I suggest you check your email, because you will receive an invitation to log in to QFieldCloud support portal where you can track your progress.

In fact, I thank you, because I have a problem with qfiled cloud and I had not noticed that there is an area to open a ticket but I read your post and realised that it could be done.

I have a qfieldcloud Pro subscription and it operated that way when I have opened the ticket. The mail tittle is "QFieldCloud Customer Self Service Portal Invitation"


I hope you solve it.

BR

Gonzalo.

**Reply by Oisin-JK** [reply]
Thanks yaqza. 
I have indeed found an email in Spam, but it seems to respond to my initial chat requests and not the ticket I submitted. I will try again.
